frontend/src/v5/ui/themes/overrides-mui.d.ts

```diff
@@ -74,3 +74,9 @@ declare module '@mui/material/styles/createTypography' {
 		kicker?: CSSProperties;
 	}
 }
+
+declare module '@mui/material/Chip' {
+	interface ChipPropsVariantOverrides {
+		text: true;
+	}
+}
```

Comment:
Is this really necessaary? If I understood it correctly you are using variants for `src\v5\ui\controls\chip\chip.component.tsx` which is not the actual MuiChip

Reply:
removed